### PR TITLE
Added SolarEdge

### PR DESCRIPTION
**Pull request by klein0r**

### CODE REVIEW

File: PV-Wechselrichter/SolarEdge/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# SolarEdge Wechselrichter SK8K
+
+---------------------------------------------------------
+
+## Weitere kompatible Sungrow Wechselrichter:
```
Comment (Apollon77) [inline PV-Wechselrichter/SolarEdge/README.md:5]: Typo? :-)

**Review by Apollon77 — CHANGES_REQUESTED** [review]
Additionally: Any special non default settings needed?

File: PV-Wechselrichter/SolarEdge/README.md (hunk 1 above)
Comment (klein0r) [reply PV-Wechselrichter/SolarEdge/README.md:5]: Copy paste 😄

**Comment by klein0r** [comment]
I would like to add another TSV file which includes further holding registers for an optional energy meter. How should I name that file?

**Comment by Apollon77** [comment]
I would say use "holding-registers-*.tsv" and add an info in the README which file contains what

**Comment by Apollon77** [comment]
Exactly, thank you!

**Pull request merged**